Lib/test/test_gettext.py

```diff
@@ -691,6 +691,39 @@ def test_cache(self):
         self.assertEqual(t.__class__, DummyGNUTranslations)
 
 
+class FallbackTranslations(gettext.NullTranslations):
+    def gettext(self, *args):
+        return 'gettext'
+
+    ngettext = pgettext = npgettext = gettext
```

Comment:
You need different return values to check that the fallback uses the right method (gettext() is not called for ngettext(), etc). You need to include arguments in the result to check that all arguments are passed correctly (not missed, not duplicated, in right order).

Reply:
Thanks for the feedback! I now test each function individually